packages/winpanda/extra/src/winpanda/docs/README.md

```diff
@@ -1,4 +1,139 @@
-# Winpanda Design Documents
+# Winpanda The DC/OS Windows Package Manager
+
+# Major features
+
+# Documentation
+
+## Command line parametres
+
+
+ ### Usage:
+ ```
+  winpanda {setup} [options]
+  winpanda {teardown} [options]
+  winpanda {start} [options]
+  winpanda {stop} [options]
+```
+
+### Commands:
+
+* Setup - This command  doing following in sequence.
+    1. Reads cluster.conf from etc folder of inst_root looks for bootstrap node configured there.
+    2. Then downloads packages.
+    3. Unpacks  to  filesystem.
+    4. Makes substitution of dcos-config-windows.yaml.
+    5. Executes extra and NSSM config files.
+* Start - Just starts MSSM and SC services.
+* Stop - Just stops all dcos services.
+* Teardown - .
+
+### Options:
+
+```
+  --target=<target>                 target operational scope for a command
+                                    (choose from: {valid_cmd_targets})
```

Comment:
No point showing the placekeeper here. Maybe copy this from the actual displayed `winpanda` usage info?